weave-js/src/components/Form/Select.tsx

```diff
@@ -92,6 +95,7 @@ export type AdditionalProps = {
   size?: SelectSize;
   errorState?: boolean;
   groupDivider?: boolean;
+  cursor?: string;
```

Comment:
So we can customize the cursor to be "text" in this case for the text input box, signifying users can type in a company name